persistence/sql/index.md

```diff
@@ -30,7 +30,7 @@ SQL Persistence can store NServiceBus data in multiple different relational data
 * Sagas are:
   * Stored using [Json.NET](http://www.newtonsoft.com/json) to serialize complex data structures, with no need to manage complex table structures
   * Built to be [version-aware](saga.md#json-net-settings-custom-settings-version-type-specific-deserialization-settings) with support for data evolution
-  * Built to [allow changing the `CorrelationId` over time](saga.md#correlation-ids-correlation-and-transitional-ids)
+  * Built to [allow changing the `CorrelationId` over time](saga.md#correlation-ids)
```

Comment:
This is just the intro I wrote awhile back to give a visitor a sense of the features made SqlP attractive before getting into the technical stuff. The URL had to change because of the complexities of having older majors (v1 which is out of support and v2 which runs out before the end of the year) that don't support core sagas. The user still gets led to the same content.